packages/api-docs-builder/ApiBuilders/ComponentApiBuilder.ts

```diff
@@ -656,7 +678,11 @@ const attachCssVariables = (reactApi: ComponentReactApi, params: ParsedProperty[
       const deprecation = deprecationTag?.text?.[0]?.text;
 
       const typeTag = propDescriptor.tags?.type;
-      const type = (typeTag?.text?.[0]?.text ?? 'string').replace(/{|}/g, '');
+
+      let type = typeTag?.text?.[0]?.text ?? defaultType;
+      if (typeof type === 'string') {
+        type = type.replace(/{|}/g, '');
+      }
```

Comment:
```suggestion
      let type = typeTag?.text?.[0]?.text ?? defaultType;
      type = type?.replace(/{|}/g, '');
```

Except if `defaultType` can be solething else than a string

Reply:
> Except if defaultType can be solething else than a string

It can be undefined (e.g. data attributes that are empty, they don't have type).